setup.py

```diff
@@ -56,11 +39,10 @@ def _data_files():
 battery status.
 '''.strip(),
 		author='Daniel Pavel',
-		author_email='daniel.pavel@gmail.com',
 		license='GPLv2',
 		url='http://pwr-solaar.github.io/Solaar/',
 		classifiers=[
-			'Development Status :: 4 - Beta',
+			'Development Status :: 5 - Production/Stable',
```

Comment:
Not sure about this.

Reply:
Me neither, but version 1 generally implies production.

Reply:
But I'll change it back and we can revisit at the next release.